ops/scripts/stats.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
```

Comment:
Perhaps we could write this to a file that gets uploaded to the artifacts if the tests fail?

```bash
while true; do
  date >> file.txt
  docker stats --no-stream >> file.txt
  sleep 10;
done
```

Reply:
Yeah for sure. For now I just wanted to see that this would even work in the first place.